src/fetch/ipfs.py

```diff
@@ -85,6 +93,12 @@ async def fetch_many(  # pylint: disable=too-many-locals
                 result = await cid.fetch_content(
                     max_retries, previous_attempts, session, first_seen_block
                 )
+                if isinstance(result, NotFoundContent):
+                    # Try Fetching the old format
+                    result = await cls.old_schema(app_hash).fetch_content(
+                        max_retries, previous_attempts, session, first_seen_block
```

Comment:
You could do it in parallel instead of only when the first try fails.

Reply:
I made an issue for this:

https://github.com/cowprotocol/dune-sync/issues/49